src/nupic/types/BasicType.hpp

```diff
@@ -68,9 +70,19 @@ class BasicType {
   // of NTA_BasicType_Int32 is "Int32"
   static NTA_BasicType parse(const std::string &s);
 
+  // convert elements of one array to another array of the specified types.

```

Comment:
PS: I think there is an automated doc system, so next time please stick to the 
/**
 *
 */ for method doc comments. You can use goodies as @param @return @throws etc